plugins/docker_binary_builder/test/models/binary_builder_test.rb

```diff
@@ -78,6 +80,14 @@
       )
     end
 
+    it 'sets up global environment variables for the build container' do
+      project.environment_variables.create!(name: 'FIRST', value: 'first')
+      project.environment_variables.create!(name: 'SECOND', value: 'second')
+      project.environment_variables.create!(name: 'THIRD', value: 'third',
+                                            scope_type: 'Environment', scope_id: environments(:production).id)
+      builder.send(:create_container_options)['Env'].must_equal %w(FIRST=first SECOND=second)
```

Comment:
That's the actual test, it should pass only global variables and ignore the ones that have scope defined :)